package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jquery-highlight",
-  "version": "3.4.0",
+  "version": "3.5.0",
```

Comment:
Could you remove this bump. I usually do this with my own process when releasing.